src/main/java/sk/uniba/fmph/dcs/player_board/PlayerBoard.java

```diff
@@ -0,0 +1,132 @@
+            final TribeFedStatus tribeFedStatus, final PlayerTools playerTools) {
+        this.playerResourcesAndFood = playerResourcesAndFood;
+        this.playerCivilisationCards = playerCivilisationCards;
+        this.playerFigures = playerFigures;
+        this.tribeFedStatus = tribeFedStatus;
+        this.playerTools = playerTools;
+    }
+
+    public void addPoints(final int points) {
+        this.points += points;
+    }
+
+    public void addHouse() {
+        houses++;
+    }
+
+    public void addEndOfGamePoints() {
+        points += playerCivilisationCards.calculateEndOfGameCivilisationCardPoints(houses, playerTools.getToolCount(),
+                tribeFedStatus.getFields(), playerFigures.getTotalFigures());
+        points += playerResourcesAndFood.numberOfResourcesForFinalPoints();
+    }
+
+    @Override
+    public String state() {
+        Map<String, String> state = Map.of(
+                "points", String.valueOf(points),
+                "houses", String.valueOf(houses),
+                "playerResourcesAndFood", playerResourcesAndFood.state(),
+                "tribeFedStatus", tribeFedStatus.state(),
+                "playerFigures", playerFigures.state(),
+                "playerCivilisationCards", playerCivilisationCards.state(),
+                "playerTools", playerTools.state()
+        );
+        return new JSONObject(state).toString();
+    }
+
+    public PlayerResourcesAndFood playerResourcesAndFood() {
+        return this.playerResourcesAndFood;
+    }
+
+    public TribeFedStatus tribeFedStatus() {
+        return this.tribeFedStatus;
+    }
+
+    public PlayerFigures playerFigures() {
+        return this.playerFigures;
+    }
+
+    public  PlayerCivilisationCards playerCivilisationCards() {
+        return this.playerCivilisationCards;
+    }
+
+    public  PlayerTools playerTools() {
+        return this.playerTools;
+    }
+
+    public void giveEffect(final Collection<Effect> stuff) {
+        for (Effect effect : stuff) {
+            switch (effect) {
+                case FOOD, WOOD, CLAY, STONE, GOLD:
+                    playerResourcesAndFood.giveResources(List.of(effect));
+                    break;
+                case TOOL:
+                    playerTools.addTool();
+                    break;
+                case FIELD:
+                    tribeFedStatus.addField();
+                    break;
+                case BUILDING:
+                    addHouse();
+                    break;
+                case ONE_TIME_TOOL2:
+                    playerTools.addSingleUseTool(ONE_TIME_TOOL2_STRENGTH);
+                    break;
+                case ONE_TIME_TOOL3:
+                    playerTools.addSingleUseTool(ONE_TIME_TOOL3_STRENGTH);
+                    break;
+                case ONE_TIME_TOOL4:
+                    playerTools.addSingleUseTool(ONE_TIME_TOOL4_STRENGTH);
+                    break;
+                case POINT:
+                    addPoints(1);
```

Comment:
```suggestion
                    addPoints(1);
                    break;
```
Inak by sme dostali výnimku pri každom `POINT`, zabudol som na to v predošlej review.